### PR TITLE
JOSS: copy edits/bib

### PR DESCRIPTION
**Pull request by jedbrown**
Some light copy edits from the [JOSS review](https://github.com/openjournals/joss-reviews/issues/6766).

The DOE notice is intended for the publisher -- JOSS acknowledges it (and JOSS license policy is always compatible with those terms).

### CODE REVIEW

**Comment by quantumsteve** [comment]
@jedbrown Our understanding is that the ORNL's notice of copyright must be included in all journals and conference papers we submit. Below are two published JOSS papers that both have ORNL's notice of copyright in the acknowledgements.

https://joss.theoj.org/papers/10.21105/joss.05345
https://joss.theoj.org/papers/10.21105/joss.00407

**Comment by jedbrown** [comment]
Hmm, in the past, these notices have been for the publisher's awareness, but not to be printed in the published version. I believe this is still the case for Argonne and LBNL (the publisher acknowledges the notice and publishes with that text removed). I see that the notice is present in the published version of your recent JCP paper. If the policy is that this must be included in the published version, it can be left in.

File: paper/paper.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ authors:
   - name: Miroslav K Stoyanov
     orcid: 0000-0002-8199-5577
     affiliation: 1
-  - name: Stefan Schanke
+  - name: Stefan Schnake
```
Comment (jedbrown) [inline paper/paper.md:18]: I assume this was just a typo.

Comment (quantumsteve) [reply paper/paper.md:18]: Thank you for catching that.

**Comment by quantumsteve** [comment]
> Hmm, in the past, these notices have been for the publisher's awareness, but not to be printed in the published version. I believe this is still the case for Argonne and LBNL (the publisher acknowledges the notice and publishes with that text removed). I see that the notice is present in the published version of your recent JCP paper. If the policy is that this must be included in the published version, it can be left in.

Yes, please bring back this notice so I can merge the rest of these edits.

File: paper/paper.md [hunk 2]
```diff
@@ -93,12 +93,10 @@ The goal of ASGarD is to facilitate and promote the use of adaptive sparse-grid
 
 # State of the Field
 
-While GitHub Topics reports 65 public repositories for the discontinuous-galerkin method and 13 for sparse-grids, only two combine these two techniques. GalerkinSparseGrids.jl is written in Julia and missing adaptivity, distributed and shared-memory parallelism or GPU accelerator support [@atanasov2017]. AdaM-DG is also written in C++ and missing distributed memory parallelism or GPU accelerator support [@huang2024].
+While GitHub Topics reports 65 public repositories for the discontinuous-Galerkin method and 13 for sparse-grids, only two combine these two techniques. GalerkinSparseGrids.jl is written in Julia and missing adaptivity, distributed- and shared-memory parallelism or GPU accelerator support [@atanasov2017]. AdaM-DG is written in C++ and lacks distributed-memory parallelism or GPU accelerator support [@huang2024].
 
 # Acknowledgements
```
Comment (quantumsteve) [inline paper/paper.md:99]:
```suggestion
Notice: This manuscript has been authored by UT-Battelle, LLC under Contract No. DE-AC05-00OR22725 with the U.S. Department of Energy.  The publisher, by accepting the article for publication, acknowledges that the U.S. Government retains a non-exclusive, paid up, irrevocable, world-wide license to publish or reproduce the published form of the manuscript, or allow others to do so, for U.S. Government purposes. The DOE will provide public access to these results in accordance with the DOE Public Access Plan (http://energy.gov/downloads/doe-public-access-plan).
```

Comment (quantumsteve) [reply paper/paper.md:99]: While ORNL prefers for the notice to be included in the final version, we can’t prevent the journal from removing it.

Comment (jedbrown) [reply paper/paper.md:99]: Thanks, I just got back to this. If you want to reinstate it, JOSS won't object, but everyone is granted these permissions due to the license that JOSS uses (JOSS only receives a non-exclusive license and it is distributed to everyone as CC BY 4.0).

**Pull request merged**